main.c

```diff
@@ -277,6 +277,7 @@ int WINAPI _tWinMain (HINSTANCE hThisInstance,
   if (use_iservice && strtod(o.ovpn_version, NULL) > 2.3 && !o.silent_connection)
     CheckIServiceStatus(TRUE);
 
+  CheckServiceStatus();	/* Check if automatic service is running or not */
```

Comment:
When automatic service is not installed (which is by default), this triggers "Failed to open OpenVPNService" message box.

Reply:
Oh, right, we have to remove that warning -- will fix.
BTW, though this is not the place to discuss it, why don't we install automatic service by default (but not set it to start unless required)? It's a pain to install it later as we do not support modify in the MSI.